cppsdk/include/playfab/PlayFabEvent.h

```diff
@@ -0,0 +1,94 @@
+#pragma once
+
+#ifndef DISABLE_PLAYFABENTITY_API
```

Comment:
Can you please have some verbage around what to expect as I'm reading this file ?